src/test/java/com/algorand/algosdk/unit/Rekeying.java

```diff
@@ -70,4 +80,31 @@ public void the_signed_transaction_should_equal_the_golden(String golden) throws
     public void i_set_the_from_address_to(String fromAddress) {
         transactionBuilder.sender(fromAddress);
     }
+
+    @Given("multisig addresses {string}")
```

Comment:
Are these related to fee pooling? Or are they just sneaking into this PR for convenience?

Reply:
these are steps for fee pooling tests, https://github.com/algorand/algorand-sdk-testing/blob/master/features/unit/feetest.feature#L29-L31. I had to put these in the Rekeying class because that's where the payment transaction builder is implemented. Cucumber doesn't allow me to duplicate the same definition in a different file.

Reply:
Gotcha - that makes sense.